examples/starcoder2/qlora.yml

```diff
@@ -0,0 +1,70 @@
+base_model: bigcode/starcoder2-3b
+
+load_in_8bit: false
+load_in_4bit: true
+strict: false
+
+datasets:
+  - path: mhenrichsen/alpaca_2k_test
+    type: alpaca
+
+chat_template: chatml
```

Comment:
```suggestion
```

Let users decide?

Reply:
agreed. plus specifying chatml here shouldn't work with the type: alpaca defined above it.